cmd/kubeadm/app/cmd/config_test.go

```diff
@@ -349,6 +353,7 @@ registry.k8s.io/coredns/coredns:{{.CoreDNSVersion}}
 	}
 }
 
+// Tests the image pulling functionality of the container runtime.
```

Comment:
we don't really need the those comments for those testcases, too much verbose and it can be easily infer from the func name I think.